resources/services/shield/protection_groups.go

```diff
@@ -0,0 +1,90 @@
+package shield
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/shield"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource protection_groups --config gen.hcl --output .
+func ProtectionGroups() *schema.Table {
```

Comment:
Missing https://docs.aws.amazon.com/waf/latest/DDOSAPIReference/API_ListResourcesInProtectionGroup.html
 -> can add in separate PR (open issue)

Reply:
protection_group has `members` column which tells which resources are in protection group